reference/v8.json

```diff
@@ -43,6 +43,37 @@
       "doc": "Default pitch, in degrees. Zero is perpendicular to the surface, for a look straight down at the map, while a greater value like 60 looks ahead towards the horizon. The style pitch will be used only if the map has not been positioned by other means (e.g. map options or user interaction).",
       "example": 50
     },
+    "light": {
+      "anchor": {
+        "type": "enum",
+        "default": "viewport",
+        "values": [
+          "map",
+          "viewport"
+        ],
+        "doc": "Whether extruded geometries are lit relative to map or viewport. The light anchor property will only be used if it has not been set by map options upon initialization.",
+        "example": "map"
+      },
+      "direction": {
+        "type": "array",
+        "value": "number",
+        "default": [1.15, 210, 30],
+        "doc": "Light direction for lighting extruded geometries, in [r radial coordinate, a azimuthal angle, p polar angle] where r indicates the distance from the center of an object to its light, a indicates the position of the light relative to 0º (like a clock), and p indicates the height of the light (from 0º, directly above, to 180º, directly below). The light direction property will only be used if it has not been set by map options upon initialization.",
```

Comment:
I think it would be more intuitive to describe this property as the position of the light source relative to the feature, rather than the lighting direction. “Lighting direction” could be confused with the direction to which the light source points, as opposed to the position from which it comes. (Or am I misunderstanding this property?)